values/kubeclarity/kubeclarity.gotmpl

```diff
@@ -0,0 +1,14 @@
+{{- $v := .Values }}
+{{- $k := $v.apps.kubeclarity }}
+
+global:
+  databasePassword: {{ $k | get "databasePassword" }}
+
+kubeclarity-postgresql:
+  fullnameOverride: kubeclarity-postgresql
+
+kubeclarity:
+  logLevel: {{ $k | get "logLevel" "warning" }}  
```

Comment:
please set such defaults in defaults.yaml

Reply:
so many words...it is giving me friction...just please unpack from now on and don't go against the workflow

Reply:
You assume waaaay to much...is what I am assuming

Reply:
Unpacking means producing a broken diff vs the original chart that **just shows file additions and not changes (if we were to make any**). Please kindly and **professionally** address this and I will be more than happy to implement.